inscriptions/ISic001400.xml

```diff
@@ -191,24 +188,33 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140908">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AR">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Θεοῖς Κατα-<lb n="2"/>γαιδίοις. ❦<lb n="3"/>Κορνελί<supplied reason="omitted">α</supplied> ❦<lb n="4"/>χρηστὴ χαῖ-<lb n="5"/>ρε· ἔζησες ❦<lb n="6"/>❦ ἔτη κβʹ.
-                    <lb n="7"/>
-                </ab>
-            </div>
+                    <lb n="1"/>Θεοῖς Κατα
+                    <lb n="2" break="no"/>γαιδίοις <g type="leaf">❦</g> 
+                    <lb n="3"/><name>Κορνελία</name> <g type="leaf">❦</g>  
+                    <lb n="4"/>χρηστὴ χαῖ
+                    <lb n="5" break="no"/>ρε ἔζησες <g type="leaf">❦</g>
+                    <lb n="6"/><g type="leaf">❦</g> ἔτη κβ
+                 </ab>
+                </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>text of IG</note>
+                    </app>
+                </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="it" resp="#AR">
+                <p>Agli dei sotterranei. Addio Cornelia, gentile e premurosa. Sei morta all'età di 22 anni</p>
+            </div>
+            <div type="translation" xml:lang="en" resp="#AR">
+                <p>To the gods who dwell beneath. Farewell Cornelia, kind and caring. You died at 22 years old</p>
```

Comment:
there's only 'χρηστη', which could be 'kind', but not caring also. As a point of principle, i would say 'you lived for 22 years', since the verb is live not die.